04/Version1/src/abstracts/_mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin respond($breakpoint) {
+  @if $breakpoint ==desktop {
+    @media (min-width: 601px) {
```

Comment:
Trochę mała ta szerokość jak na desktop, ale rozumiem, że to tylko na potrzeby tego zadania :P